tests/e2e/etcd_mix_versions_test.go

```diff
@@ -27,24 +27,13 @@ import (
 	"go.etcd.io/etcd/tests/v3/framework/e2e"
 )
 
-// TestMixVersionsSendSnapshot tests the mix version send snapshots
-// TODO(ahrtr): add network partition scenario to trigger snapshots.
-func TestMixVersionsSendSnapshot(t *testing.T) {
+// TestMixVersionsSnapshotByAddingMember tests the mix version send snapshots by adding member
+func TestMixVersionsSnapshotByAddingMember(t *testing.T) {
 	cases := []struct {
 		name               string
 		clusterVersion     e2e.ClusterVersion
 		newInstanceVersion e2e.ClusterVersion
 	}{
-		// etcd doesn't support adding a new member of old version into
-		// a cluster with higher version. For example, etcd cluster
-		// version is 3.6.x, then a new member of 3.5.x can't join the
-		// cluster. Please refer to link below,
-		// https://github.com/etcd-io/etcd/blob/3e903d0b12e399519a4013c52d4635ec8bdd6863/server/etcdserver/cluster_util.go#L222-L230
-		/*{
-			name:              "etcd instance with last version receives snapshot from the leader with current version",
-			clusterVersion:    e2e.CurrentVersion,
-			newInstaceVersion: e2e.LastVersion,
-		},*/
```

Comment:
Please keep this so that other contributors can understand why we did not add such sub case.

Reply:
Done